config/env_sample.json

```
@@ -89,17 +89,17 @@
     "STUDENT_DASHBOARD_LTI": false,
     "/*LTI 1.3 configuration":"*/",
     "LTI_CONFIG": {
-        "/*canvas LMS domain nanme":"*/",
+        "/*canvas LMS domain name":"*/",
         "https://canvas.test.instructure.com": [
```

I think @zqian meant for this line, `"https://canvas.test.instructure.com"`, to be changed.
```suggestion
        "https://umich.test.instructure.com": [
```